tuxemon/core/components/event/actions/npc.py

```diff
@@ -70,20 +71,21 @@ def create_npc(self, game, action):
             return
 
         # Get the npc's parameters from the action
-        name = str(action.parameters[0])
-        tile_pos_x = int(action.parameters[1])
-        tile_pos_y = int(action.parameters[2])
-        animations = str(action.parameters[3])
-        behavior = str(action.parameters[4])
+        slug = str(action.parameters[0])
+        name = str(action.parameters[1])
```

Comment:
The name field...I think the NPC class should get the name from the json db, rahter than relying on the map to supply a name.  Localization and all.